docs/references/configuration.rst

```diff
@@ -599,11 +599,17 @@ jwt-aud
   **In-Database** pgrst.jwt_aud
   =============== =================================
 
-  Specifies the `JWT audience claim <https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.3>`_. If this claim is present in the client provided JWT then you must set this to the same value as in the JWT, otherwise verifying the JWT will fail.
+  Specifies the `JWT audience claim <https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.3>`_.
+
+  By default, PostgREST identifies with all audiences and allows the JWT for any ``aud`` claim.
+
+  Once this config is set, PostgREST will compare this value to the one in the JWT ``aud`` claim.
+  If this comparison fails, then the token is rejected.
 
   .. warning::
 
-     Using this setting will only reject tokens with a different audience claim. Tokens **without** audience claim will still be accepted.
+     Using this setting will only reject tokens with a different audience claim.
+     Tokens will still be accepted if the ``aud`` claim **is not present** or if its value is ``null`` or ``[]``.
```

Comment:
Instead of a warning note, I think we should structure the `aud` rules as bulletpoints. Also taking into account the above feedback. That'll be more complete and explicit.

Reply:
Also, I think these rules would make more sense in a subsection called `"aud validation"` in `auth.rst`, not on the config page.